<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="ui_exec_btn">
        <property name="text">
         <string>Exec Dialog</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="ui_show_btn">
        <property name="text">
         <string>Show Dialog</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="ui_count_btn">
        <property name="text">
         <string>Show Count</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="ui_test_menu">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="ui_exec_act"/>
    <addaction name="ui_show_act"/>
    <addaction name="ui_count_act"/>
   </widget>
   <addaction name="ui_test_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ui_exec_act">
   <property name="text">
    <string>Exec Dialog</string>
   </property>
  </action>
  <action name="ui_show_act">
   <property name="text">
    <string>Show Dialog</string>
   </property>
  </action>
  <action name="ui_count_act">
   <property name="text">
    <string>Show Count</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
